src/Xamarin.Android.Linker/Xamarin.Android.Linker.csproj

```diff
@@ -0,0 +1,55 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\Configuration.props" />
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <DefineConstants>NET5_LINKER</DefineConstants>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <OutputPath>$(XamarinAndroidSourcePath)bin\Build$(Configuration)\packs\tools</OutputPath>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.ILLink" Version="5.0.0-preview.3.20227.3" />
+    <ProjectReference Include="..\..\external\Java.Interop\src\Java.Interop.Export\Java.Interop.Export.csproj" />
```

Comment:
I do not understand why we would have a `@(ProjectReference)` to `Java.Interop.Export.dll`.  I don't see any mentions of types located within that assembly within this PR, e.g. neither `JniSignature` nor `MarshalMemberBuilder` make an appearance.

Reply:
This is to have `MarshalMemberBuilder.GetMarshalMethodName` for `TryGetMarshalMethod` method.